openapi.yaml

```diff
@@ -2397,6 +2397,43 @@ paths:
           $ref: '#/components/responses/client_error_auth'
         5XX:
           $ref: '#/components/responses/server_error'
+  /processing_options:
+    get:
+      summary: Additional processing options
+      operationId: list-processing-options
+      description: |-
+        Lists additional custom processing options that a back-end offers for the
+        different processing modes (synchronous processing, batch jobs, secondary web services).
+
+        The options specified here can be added to the corresponding `POST` requests
+        at the top-level of the object that is sent as the payload.
+        All options SHOULD explicitly be made optional with reasonable defaults
```

Comment:
Couple of slightly related notes/questions:

- are the defaults global or can they depend on user (e.g. subscription level or some other property)?
- is it allowed to use a default like `null` in the public parameter specification which allows the backend to use a an actual fallback value that is not public or dynamic in some sense (e.g. user dependent, dependent on current cluster load, automatically derived from  spatial/temporal extent, ...)?
- is it necessary to cover these concerns in the description?

Reply:
I think that's all up to the implementation to decide.